src/bigcommerce.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable eslint-comments/no-unlimited-disable */
+/* eslint-disable */
+
+if (window.SGJavascriptBridge) {
+  shopgateInit();
+} else if (navigator.userAgent.indexOf('libshopgate') !== -1) { // Shopgate iOS App
```

Comment:
Would you consider early return here also?

Reply:
done